### PR TITLE
refactor(backup, pdc)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Refactoring of backup and pdc related code.

### Added

-

### Changed
- Renamed `BackupAPI` to `IlluminaBackupService`
- Renamed `PdcAPI` to `PdcService`
- Moved flow cell specific backup functions from the `PdcService` to the `IlluminaBackupService`
- Moved tests
- Moved fixtures

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/services/pdc_service/pdc_service.py [hunk 1]
```diff
@@ -86,57 +86,6 @@ def run_dsmc_command(self, command: list) -> None:
...
-        )
-        self.backup_flow_cell(
-            files_to_archive=[
-                flow_cell_encryption_api.final_passphrase_file_path,
-                flow_cell_encryption_api.encrypted_gpg_file_path,
-            ],
-            store=status_db,
-            db_flow_cell=db_flow_cell,
-        )
-
```
Comment (ChrOertlin) [inline cg/services/pdc_service/pdc_service.py:139]: most important part here: moving of these backup related frunctions out of the pdc service. So that it only is used for pdc related commands

Comment (ChrOertlin) [reply cg/services/pdc_service/pdc_service.py:139]: The `IlluminaBackupService` should make these function calls and make use of the pdc service.

File: cg/constants/archiving.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from enum import StrEnum
 
-DEFAULT_SPRING_ARCHIVE_COUNT = 100
+DEFAULT_SPRING_ARCHIVE_COUNT = 500
```
Comment (diitaz93) [inline cg/constants/archiving.py:3]: what is the purpose of this change?

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

**Review by Vince-janv — APPROVED** [review]
Nice! Logic looks intact ⭐

File: cg/constants/archiving.py (hunk 2 above)
Comment (Vince-janv) [reply cg/constants/archiving.py:3]: This looks like an artefact of branching from master instead of the feature branch. Isak just merged a PR changing it from 100 to 500 in master

Comment (ChrOertlin) [reply cg/constants/archiving.py:3]: Yes, indeed.
